Source/Dafny/DafnyAst.cs

```diff
@@ -6760,6 +6781,7 @@ public UpdateStmt(IToken tok, IToken endTok, List<Expression> lhss, List<Assignm
   public class AssignOrReturnStmt : ConcreteUpdateStatement
   {
     public readonly Expression Rhs; // this is the unresolved RHS, and thus can also be a method call
+    public readonly IToken ExpectToken; // TODO-RS: Implement!
```

Comment:
What does the comment mean? What is it that's not implemented?

Reply:
Leftover from the "let's compile assumes" partial work, will remove.